frontend/public/components/RBAC/bindings.jsx

```diff
@@ -101,8 +104,8 @@ export const RoleLink = ({binding}) => {
 };
 
 const Row = ({obj: binding}) => <ResourceRow obj={binding}>
-  <div className="col-md-3 col-sm-4 col-xs-6 co-resource-link-wrapper">
```

Comment:
Can this class be removed from the CSS now, or is it still used somehwere?

Reply:
It's used in three Cogs and a ResourceLink (in silence.tsx, alert.tsx, cluster-service-class.tsx, and cluster-service-plan.tsx). I only updated ResourceCogs. Let me see if I can track them down and see what they look like.

Reply:
@spadgett I don't have access to /monitoring/silences or /monitoring/alertrules on the shared cluster, and I don't think ClusterServiceClassPage is visible because OLM isn't enabled. Is there a different way to check these pages?

Reply:
Cluster Service Classes are under the Service Catalog nav.

The silence and alert rule pages need a newer version of monitoring than we have installed, so that will be hard to check.